From f043b6758be4e24c2b490b5d228b50d7cc318106 Mon Sep 17 00:00:00 2001
From: Frank Wunderlich <frank-w@public-files.de>
Date: Thu, 27 Jun 2024 22:48:27 +0200
Subject: [PATCH 005/116] arm64: dts: mt7622: fix sata

[    2.480551] ahci-mtk 1a200000.sata: missing phy-mode phandle
[    2.486249] ahci-mtk 1a200000.sata: probe with driver ahci-mtk failed with error -22

Fixes: 3ba5a6159434 ("arm64: dts: mediatek: mt7622: fix clock controllers")
---
 arch/arm64/boot/dts/mediatek/mt7622.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/mediatek/mt7622.dtsi b/arch/arm64/boot/dts/mediatek/mt7622.dtsi
index 917fa39a74f8..f22983c33365 100644
--- a/arch/arm64/boot/dts/mediatek/mt7622.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt7622.dtsi
@@ -790,7 +790,7 @@ u2port1: usb-phy@1a0c5000 {
 	};
 
 	pciesys: clock-controller@1a100800 {
-		compatible = "mediatek,mt7622-pciesys";
+		compatible = "mediatek,mt7622-pciesys","syscon";
 		reg = <0 0x1a100800 0 0x1000>;
 		#clock-cells = <1>;
 		#reset-cells = <1>;
-- 
2.39.5

